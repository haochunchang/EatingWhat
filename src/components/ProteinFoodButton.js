import React, { Fragment } from "react";

const ProteinFoodButton = ({ onClicked }) => (
  <Fragment>
    <button
      type="button"
      onClick={onClicked}
      className="f3 pa3 btn btn-sucess grow"
    >
      Protein Food?
    </button>
  </Fragment>
);

export default ProteinFoodButton;
