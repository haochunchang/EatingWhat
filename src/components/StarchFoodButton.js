import React, { Fragment } from "react";

const StarchFoodButton = ({ onClicked }) => (
  <Fragment>
    <button
      type="button"
      onClick={onClicked}
      className="f3 pa3 btn btn-sucess grow"
    >
      Starchy Food?
    </button>
  </Fragment>
);

export default StarchFoodButton;
