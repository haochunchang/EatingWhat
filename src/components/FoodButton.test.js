import React from "react";
import { shallow } from "enzyme";
import StarchFoodButton from "./StarchFoodButton";
import ProteinFoodButton from "./ProteinFoodButton";

describe("FoodButton", () => {
  it("matches StarchFoodButton snapshot", () => {
    expect(shallow(<StarchFoodButton />)).toMatchSnapshot();
  });
  it("matches ProteinFoodButton snapshot", () => {
    expect(shallow(<ProteinFoodButton />)).toMatchSnapshot();
  });
});
