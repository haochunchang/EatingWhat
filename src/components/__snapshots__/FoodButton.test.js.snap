// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FoodButton matches ProteinFoodButton snapshot 1`] = `ShallowWrapper {}`;

exports[`FoodButton matches StarchFoodButton snapshot 1`] = `ShallowWrapper {}`;
